### PR TITLE
support for event table list in plotting page

### PR DESCRIPTION
**Pull request by FBRTMaka**
@DanielJMaher , @Bobfrat 

Bob, may need to go over with you to make sure that the event overlay on the plot working right.

### CODE REVIEW

**Comment by FBRTMaka** [comment]
DO NOT MERGE YET

**Comment by FBRTMaka** [comment]
OK TO MEREGE!!!

File: ooiservices/app/uframe/assetController.py [hunk 1]
```diff
@@ -204,6 +205,44 @@ def _associate_events(id):
...
+                platform = row['referenceDesignator']['subsite']
+            if row['referenceDesignator']['node'] is not None:
+                mooring = row['referenceDesignator']['node']
+            if row['referenceDesignator']['sensor'] is not None:  
+                instrument = row['referenceDesignator']['sensor']
+            concat_ref_des =  '-'.join([platform, mooring, instrument])
+            if concat_ref_des ==  ref_des:
+                temp_dict['id'] = row['eventId']
+                temp_dict['ref_des'] = concat_ref_des
+                temp_dict['start_date'] = num2date(float(row['startDate'])/1000, units='seconds since 1970-01-01 00:00:00', calendar='gregorian')
```
Comment (FBRTMaka) [inline ooiservices/app/uframe/assetController.py]: @Bobfrat here's how the date comes out of the assetController now, basically what you had

File: ooiservices/app/uframe/plot_tools.py [hunk 2]
```diff
@@ -87,7 +88,7 @@ def plot_time_series(self, fig, is_timeseries, ax, x, y, fill=False, title='', x
         if events:
             y = ax.get_ylim()
             for event in events['events']:
-                time = num2date(float(event['start_date'])/1000, units='seconds since 1970-01-01 00:00:00', calendar='gregorian')
+                time = datestr2num(event['start_date'])
```
Comment (FBRTMaka) [inline ooiservices/app/uframe/plot_tools.py]: @Bobfrat and then changed this to give back the correct object for the 'time' container

**Comment by DanielJMaher** [comment]
@FBRTMaka Will this effect Bob's plot at all?

**Comment by FBRTMaka** [comment]
@DanielJMaher Yes.

**Comment by DanielJMaher** [comment]
@Bobfrat Do you have some time to assess and fix?

**Comment by FBRTMaka** [comment]
@DanielJMaher , this is the fix...

**Comment by Bobfrat** [comment]
:+1:  Looks good

**Comment by DanielJMaher** [comment]
@FBRTMaka I'm confused... Wasn't it working before?  

Let me step back.. If I merge this, does it break Bob's stuff?

**Comment by DanielJMaher** [comment]
Whatever.  I trust.

**Comment by Bobfrat** [comment]
big mistake!

**Comment by FBRTMaka** [comment]
lol - So yeah, this effects the plot but only to fix the update that was done to the way the events are being delivered.  So, it really effects it positively and not negatively.

**Comment by FBRTMaka** [comment]
@DanielJMaher it won't break it.

**Pull request merged**